main/ServiceManager/ServiceManager.cpp

```diff
@@ -56,7 +58,10 @@ void MatterServiceRunner()
     MatterInterfaceHandler.SharedSemaphore = &MatterSemaphore;
     MatterInterfaceHandler.MatterAttributeUpdateCB = MatterAttributeUpdateCBMain;
     MatterInterfaceHandler.ConnectToMatterNetwork = MatterNetworkConnected;
-    Matter_TaskInit(&MatterInterfaceHandler);
+    Matter_TaskInit(&MatterInterfaceHandler, 
```

Comment:
you should fill and use `ServiceManger.tasks[MatterTask]` structure to pass parameters through the `ServiceManager`
see `void GUI_TaskCreator()` at this line for reference: [https://github.com/DoneTech-IoT/IoTPlatform/blob/1a7e7fc1ff34739694ed46bdf632a25608d0cd65/main/ServiceManager/ServiceManager.cpp#L68-L69](url)

Reply:
Yea, It will change in other PR. when the ServiceManager Struct is defined more reasonably.